Core/HLE/AtracCtx.cpp

```diff
@@ -919,7 +920,12 @@ int Atrac::RemainingFrames() const {
 
 	if ((bufferState_ & ATRAC_STATUS_STREAMED_MASK) == ATRAC_STATUS_STREAMED_MASK) {
 		// Since we're streaming, the remaining frames are what's valid in the buffer.
-		return bufferValidBytes_ / track_.bytesPerFrame;
+		int RemainingFramesValue = bufferValidBytes_ / track_.bytesPerFrame;
+		if (loopNum_ == -99) {
+			loopNum_ == -1;
```

Comment:
Correct should be loopNum_ = -1. But compiler does not allow me to do it

Reply:
It's because `RemainingFrames()` is marked with const. You can remove that (both cpp and header).

However, if we should change loopnum like this, maybe this is not the right place to do it?